### PR TITLE
ai-missions structure for public repo FAB-3568

### PR DESCRIPTION
**Pull request by bpandey-CS**
added readme and Makefile for skills

ignoring commons in skills

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@
 .vscode
 venv/
 dist
+ai-missions/*/skills/*/src/common
```
Comment (pkandarpa-cs) [inline .gitignore:8]: why is this ignored?

Comment (bpandey-CS) [reply .gitignore:8]: we copy commos as part of the build (for individual skills) here -> https://github.com/CognitiveScale/cortex-fabric-examples/blob/1393fb9708a227f85bdcb920a04719122c6b0100/ai-missions/high-risk-flu-shot-mission/skills/send_virtual_care_notification/Makefile#L12-L17

after `make build` -> there would be a bunch of new files in the skills folder (that we don't want git to track, it's already there in commons)

File: ai-missions/high-risk-flu-shot-mission/skills/send_nurseline_notification/Readme.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+### Send Nurseline Notification
+
+
+#### Description:
+For the given profile IDs will send notifications on preferred communication channel to call nurse line to know more information about flushot and flushot providers.
```
Comment (smichalski-cs) [inline ai-missions/high-risk-flu-shot-mission/skills/send_nurseline_notification/Readme.md:5]: Missing some words I think.  Change to:  For the Profile Ids derived from the configured Cohort, notifications are sent on the preferred communication channel that request that members call the nurse-line for more information about scheduling a flu shot with an available flu shot provider.

File: ai-missions/high-risk-flu-shot-mission/skills/send_schedule_appointment_notification/Readme.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+### Send Schedule Appointment Notification
+
+
+#### Description:
+For the given profile IDs, this skill sends notifications on preferred communication channel to schedule a telehealth appointment.
```
Comment (smichalski-cs) [inline ai-missions/high-risk-flu-shot-mission/skills/send_schedule_appointment_notification/Readme.md:5]: Change to: For the Profile IDs derived from the configured Cohort, notifications are sent on the preferred communication channel that request that members schedule a telehealth appointment.

File: ai-missions/high-risk-flu-shot-mission/skills/update_member_phone_number/Readme.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+### Update member phone number
+
+#### Description:
+For the given profile IDs will send requests to update phone number .
```
Comment (smichalski-cs) [inline ai-missions/high-risk-flu-shot-mission/skills/update_member_phone_number/Readme.md:4]: Change to: For the Profile IDs derived from the configured Cohort, requests are sent to members to update their phone numbers.

File: ai-missions/high-risk-flu-shot-mission/skills/send_virtual_care_notification/Readme.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+### Send Virtualcare Notification
+
+
+#### Description:
+Shares given profile details with Virtual care nurse to call and inform regarding flu-shots.
```
Comment (smichalski-cs) [inline ai-missions/high-risk-flu-shot-mission/skills/send_virtual_care_notification/Readme.md:5]: For the Profile IDs derived from the configured Cohort, member contact information is shared with virtual care nurses, who call members to provide information and schedule flu-shots.

File: ai-missions/high-risk-flu-shot-mission/skills/send_engage_sydneycare_chatbot_notification/Readme.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
+### Send Sydneycare Chatbot Notification
+
+#### Description:
+For the given profile IDs, this skill sends notifications to members for engaging with a chatbot in SydneyCare to discuss flu shot options.
```
Comment (smichalski-cs) [inline ai-missions/high-risk-flu-shot-mission/skills/send_engage_sydneycare_chatbot_notification/Readme.md:4]: Change to: For the Profile IDs derived from the configured Cohort, notifications are sent to members instructing them to engage with a chatbot in the SydneyCare app to discuss flu shot options.

**Review by smichalski-cs — APPROVED** [review]
Please make the handful of wording changes requested before merging.

**Comment by bpandey-CS** [comment]
@ljha-CS this is the [commit](https://github.com/CognitiveScale/cortex-fabric-examples/pull/62/commits/46e46445189a4e0d6417ea3f6d4edc81486a0e05) for windows bat file support

File: ai-missions/high-risk-flu-shot-mission/skills/Readme.md [hunk 7]
```diff
@@ -29,19 +29,47 @@ These skills will send flu-shot reminder notifications along with provider info
...
-## Deployment steps
-
-Set the evironment variable `PROJECT_NAME` using `export PROJECT_NAME=<project_name>` and `DOCKER_PRIV_REGISTRY_URL` using `export DOCKER_PRIV_REGISTRY_URL=<project_name>`, make sure to have access to the private registry being used, for cortex users it would be as simple as `cortex docker login`.
+#### Steps to build and deploy
+
+Set environment variables `DOCKER_PREGISTRY_URL` (like <docker-registry-url>/<namespace-org>) and `PROJECT_NAME` (Cortex Project Name), and use build scripts to build and deploy.
+
+Configure Docker auth to the private registry:
+1. For Cortex DCI with Docker registry installed use `cortex docker login`
+2. For external Docker registries like Google Cloud's GCR etc use their respective CLI for Docker login
```
Comment (pkandarpa-cs) [inline ai-missions/high-risk-flu-shot-mission/skills/Readme.md:38]: Mention ECR, ACR and GCR

File: ai-missions/high-risk-flu-shot-mission/skills/make.bat [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+    for %%x in %SKILL_NAME% do ( cd "%%x" & cmd /c make deploy & cd ..)
+    GOTO :EOF
+
+:tests
+    for %%x in %SKILL_NAME% do ( cd "%%x" & cmd /c make tests & cd ..)
+    GOTO :EOF
+
+:get
+    for %%x in %SKILL_NAME% do ( cd "%%x" & cmd /c make get & cd ..)
+    GOTO :EOF
```
Comment (pkandarpa-cs) [inline ai-missions/high-risk-flu-shot-mission/skills/make.bat:25]: newline at the last line

File: ai-missions/high-risk-flu-shot-mission/skills/send_engage_sydneycare_chatbot_notification/make.bat [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+SET IMAGE_TAG=latest
+SET DOCKER_IMAGE=sc-chatbot
+SET DOCKER_IMAGE_URL=%DOCKER_PREGISTRY_URL%/%DOCKER_IMAGE%:%IMAGE_TAG%
+
+GOTO :checkenv %*
+
+
+:build
+	echo Building %DOCKER_IMAGE_URL% action image...
+    xcopy /s ..\..\common .\src
```
Comment (pkandarpa-cs) [inline ai-missions/high-risk-flu-shot-mission/skills/send_engage_sydneycare_chatbot_notification/make.bat:12]: check the indentation for most of the targets in this file

File: ai-missions/high-risk-flu-shot-mission/skills/send_nurseline_notification/make.bat [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+SET DOCKER_IMAGE_URL=%DOCKER_PREGISTRY_URL%/%DOCKER_IMAGE%:%IMAGE_TAG%
+
+GOTO :checkenv %*
+
+
+:build
+	echo Building %DOCKER_IMAGE_URL% action image...
+    xcopy /s ..\..\common .\src
+    cmd /c docker build -t %DOCKER_IMAGE_URL% .
+	echo done building %DOCKER_IMAGE% action image
```
Comment (pkandarpa-cs) [inline ai-missions/high-risk-flu-shot-mission/skills/send_nurseline_notification/make.bat:14]: check indentation in this file

**Pull request merged**